project/plugins.sbt

```diff
@@ -1,8 +1,8 @@
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.9")
```

Comment:
This was needed because the test refactors broke the assembly step - they got tripped up on valid JVM 11 byte code.  Turns out the first version of this with working JVM 11 support was 0.14.9 and we were just getting lucky it wasn't breaking before.